apps/marketplace/components/BuyerTraining/BuyerTrainingSelectStage.js

```diff
@@ -92,6 +92,7 @@ export class BuyerTrainingSelectStage extends Component {
           <div className="col-xs-12 col-sm-9">
             <div className={styles.selectSellers}>
               <SellerSelect
+                briefId={this.props[model].id}
```

Comment:
Just comparing RFX and Specialist. I notice you use 
`briefId={this.props[this.props.model].id}` is that the case for Training as well?

Reply:
This one is different because `model` was destructured on line [51](https://github.com/AusDTO/dto-digitalmarketplace-frontend/pull/964/files#diff-7c0888cafab060c51a02e80153a25b4cR51).